### DISCUSSION TITLE
getServerSideProps context.params is rewritten when request is made client side

### DISCUSSION THREAD

**Discussion by connershoop** [vercel/next.js#13566] [Help]
# Bug report

## Describe the bug
In my `getServerSideProps` function, `context.params` is overwritten and the function reruns. When the page makes a request (ex: `/static/image.png`) stemming from `<img src='/static/image.png' />` the context.params in my `getServerSideProps `are overwritten to `{param1name: 'static', param2name: 'image.png'}`

## To Reproduce
Reproduced here
https://github.com/connershoop/Nextjs-getServerSideProps-Issue

## Expected behavior

`getServerSideProps` only runs once, and `context.params` remains the path parameters

## System information

- OS: [e.g. PopOS]
- Browser (if applies) [e.g. chrome]
- Version of Next.js: [e.g. 9.3.5]
- Version of Node.js: [e.g. 12.16.3]

**Comment by connershoop** [comment]
This is an improper way to get images, they should be placed in the public folder. But I feel this is still a relevant issue

**Comment by connershoop** [comment]
Seems like this is only happening in chrome from other threads: https://github.com/vercel/next.js/discussions/13064

**Comment by LeOndaz** [comment]
did you solve this?
